### DISCUSSION TITLE
Choosing a country and language when setting up a BitClout account.

### DISCUSSION THREAD

**Discussion by 54rrdeaniel** [deso-protocol/dips#158] [Medium CIPs]
Before the Referral Reward program, we need the following feature.

💎 Choosing a country and language when setting up a BitClout account.

If we have this data of users, we can create a specific feed for each country. You know, not everyone in the world knows English very well. Therefore, users who do not speak English should also have their own Feed. In this way, it will be easier to create local communities.
And Local Node owners will be able to make more effective developments using this data.

With the Referral Rewards Program, users will set up Bitclout accounts. But these users need to feel belonging here in order to use this site continuously. A person who does not speak English can enter this site a maximum of 5 more times. It would definitely be better if newcomers see a Local Feed where their language is spoken. And they will start to share content so that their posts can be included here.

**Comment by darian-parrish** [comment]
This is solved by using https://desocialworld.com/
